src/components/LastModifiedActorCoreUnit/LastModifiedActorCoreUnit.tsx

```diff
@@ -16,7 +16,7 @@ interface Props {
   className?: string;
 }
 
-export const ActorLastModified = ({ date, now = DateTime.now(), href, className }: Props) => {
+export const LastModifiedActorCoreUnit = ({ date, now = DateTime.now(), href, className }: Props) => {
   const isDesk = useMediaQuery(lightTheme.breakpoints.up('desktop_1024'));
```

Comment:
We should update things like this while we apply the reskin to keep the same code styles, here, we should use a function callback with the theme instead of using the `lightTheme`

```tsx
const isDesk = useMediaQuery((theme: Theme) => theme.breakpoints.up('desktop_1024'));
```

Reply:
done